src/main/java/core/basesyntax/utils/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.utils;
+
+public interface DrawFigure {
```

Comment:
better interface name. google about java interface naming advices
```suggestion
public interface Drawable {
```